All_materials/20230822_Web_Scraping_Financial_Ombudsman_Service/config.toml

```diff
@@ -0,0 +1,18 @@
+project_name = "fos_web"
+
+[web]
+seach_term = "scam"
+start_date = "2023-06-01"
+end_date = "2023-06-02"
+
+[table]
+ref = 'Ref'
```

Comment:
make sure you use consistent string notation, e.g. ' or "

Reply:
done